README.md

```diff
@@ -711,6 +711,19 @@ end
 
 ```
 
+You may wish to have elements declared in a page object class that are not always guaranteed to be present (success or error messages, etc.). If you'd still like to test such a page with `all_there?` you can define an `excluded_elements` method on your page object class that excludes the specified elements from the `all_there?` check.
+
+```ruby
+class TestPage < SitePrism::Page
+  element(:name_field, '#name')
```

Comment:
The style in (most of?) the rest of the documentation does not use parentheses around these arguments.  If you have time maybe change it to be consistent.

e.g. `  element :search_field, "input[name='q']"`

Reply:
That conflicts with prior review input from @luke-hill that explicitly asks for the parentheses to be used.

Reply:
Right, my bad.  I didn't know he was trying to change that.  I commented his comment.